tests/testAutopas/tests/selectors/FeatureVectorTest.cpp

```diff
@@ -128,14 +131,15 @@ TEST_F(FeatureVectorTest, clusterEncode) {
   std::vector<Newton3Option> newtonsVec(newtons.begin(), newtons.end());
 
   FeatureVectorEncoder encoder(allCompatibleContainerTraversalEstimators, dataLayoutsVec, newtonsVec,
-                               NumberSetFinite<double>({cellSizeFactor}));
+                               NumberSetFinite<double>({cellSizeFactor}),
+                               autopas::NumberSetFinite<int>(std::set<int>({5, 15, 30})));
 
   // generate all possible combinations
   std::vector<FeatureVector> vecList;
   for (const auto &[container, traversal, estimator] : allCompatibleContainerTraversalEstimators) {
     for (const auto &dataLayout : dataLayouts) {
       for (const auto &newton3 : newtons) {
-        vecList.emplace_back(container, cellSizeFactor, traversal, estimator, dataLayout, newton3);
+        vecList.emplace_back(container, cellSizeFactor, traversal, estimator, dataLayout, newton3, 5);
```

Comment:
This should not be a set just consisting of `5`. Please emplace into `vecList` the other rebuild frequencies that you allow, i.e. 15 & 30. Alternatively, don't test multiple rebuild frequencies - but in this case, make only a rebuild frequency of 5 allowed on line 135.